R/estimate_secondary.R

```diff
@@ -177,6 +183,9 @@ estimate_secondary <- function(reports,
   ## fill any early data up
   secondary_reports[, confirm := nafill(confirm, type = "nocb")]
 
+  # Ensure that reports and secondary_reports are aligned
+  reports <- merge.data.table(reports, secondary_reports, by = "date")
```

Comment:
```suggestion
  reports <- merge.data.table(
    reports, secondary_reports[, list(date)], by = "date"
  )
```
Some tests are failing because there are some additional columns now - I think this will fix it.